### PR TITLE
add run_pylint.sh and fix new warnings

### PR DESCRIPTION
**Pull request by YifeiZuo01**
This PR add a pylint sh and fix some new warnings, and will bring up CI for pylint later,

### CODE REVIEW

**Comment by YifeiZuo01** [comment]
now the pylint warning is 0. and after review and merged, we can configure CI.

File: mbed_flasher/flashers/enhancedserial.py [hunk 1]
```diff
@@ -1,23 +1,15 @@
-#!/usr/bin/env python
```
Comment (juhhov) [inline mbed_flasher/flashers/enhancedserial.py]: don't remove shebang line

File: mbed_flasher/main.py [hunk 2]
```diff
@@ -1,19 +1,5 @@
-#!/usr/bin/env python
```
Comment (juhhov) [inline mbed_flasher/main.py]: don't remove shebang line

File: run_pylint.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+pylint --rcfile=.pylintrc --output-format=parseable ./setup.py ./mbed_flasher
```
Comment (juhhov) [inline run_pylint.sh]: include test-folder also

Comment (YifeiZuo01) [reply run_pylint.sh]: fixed

**Pull request merged**